handbook/data.md

```diff
@@ -275,6 +275,45 @@ The following is the description of the air pollution exposure data.
 ```
 `````
 
+### PM2.5 US High Resolution Grid, 2000-2016
+
+`````{dropdown} PM2.5 US Grid
+```{list-table}
+:header-rows: 0
+
+* - spatial_coverage
+  - US
+* - spatial_resolution
+  - 1km x 1km
+* - temporal_coverage
+  - 2000-2016
+* - temporal_resolution
+  - annually
+* - size
+  - ~80 MB/year
```

Comment:
You can see the exact size in the folder on fasse

Reply:
The size slightly changes year by year. Should I use the folder size instead? The point is that there is also a 10GB file in the folder, which I am not documenting here.